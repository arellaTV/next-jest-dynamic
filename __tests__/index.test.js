/* eslint-env jest */

import React from "react";
import { render } from "@testing-library/react";

import Index from "../pages/index.js";

test("Should match Snapshot", () => {
  const { asFragment } = render(<Index />);

  expect(asFragment()).toMatchSnapshot();
});
