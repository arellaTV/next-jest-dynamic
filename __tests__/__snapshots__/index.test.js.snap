// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match Snapshot 1`] = `
<DocumentFragment>
  <div>
    <p>
      Index page content
    </p>
  </div>
</DocumentFragment>
`;
